README.md

```diff
@@ -9,11 +9,11 @@ If you are a MongoDB Enterprise customer, or need Enterprise features such as Ba
 Here is a talk from MongoDB Live 2020 about the Community Operator:
 * [Run it in Kubernetes! Community and Enterprise MongoDB in Containers](https://www.youtube.com/watch?v=2Xszdg-4T6A&t=1368s)
 
-> **Note** 
-> 
-> Hi, I'm Dan Mckean 👋 I'm the Product Manager for MongoDB's support of Kubernetes. 
```

Comment:
Since you touched this - this looks very awkward in the README. Could we rewrite this part into something less personal and speak in the name of the entire team here?